README.md

```diff
@@ -36,7 +36,7 @@ pip install wazp
 Once installed, launch the dashboard by running the following command from the root of the repository:
```

Comment:
Actually, once installed `python -m wazp.app` should work from anywhere...?

Reply:
Ah no, but I think only because we need the `sample_project`. Depending on the resolution of #33 we may be able to relax these instructions.

Reply:
`start_wazp_server.sh` exists. Could rename it to ... something else without the `.sh` and put it into the `PATH`.

Same problem of needing `sample_project` (for now, at least) but yeah, perhaps that's what we want.

Reply:
ticket? 😬

Reply:
Would it not be easier to add something like this to `pyproject.toml`, then it can just be run with `wazp` without needing another file. or adding anything to the path manually?
```
[project.scripts]
wazp = "wazp.app:run"
```

Reply:
Yes, I think you can have that. No reason afaik for a script over a `[project.scripts]`.

Same problem with the sample data...

> ticket? 😬

It's because the dashboard is loading the sample pose-estimation results. Which will probably become obsolete with #33 or if `04_dashboard.py` didn't load example data (which I guess it won't eventually). Can ticket if want ticket.